lib/nerves_hub_link/configurators/local_cert_key.ex

```diff
@@ -27,7 +27,14 @@ defmodule NervesHubLink.Configurator.LocalCertKey do
   end
 
   defp maybe_add_cacerts(%{ssl: ssl} = config) do
-    %{config | ssl: Keyword.put_new(ssl, :cacerts, Certificate.ca_certs())}
+    ssl =
+      if ssl[:cacerts] || ssl[:cacertfile] do
+        ssl
+      else
+        Keyword.put_new(ssl, :cacerts, Certificate.ca_certs())
+      end
+
+    %{config | ssl: ssl}
```

Comment:
Since we're already checking both keys, there would be no need to `Keyword.put_new` and we can directly replace. That could simplify just a bit, but not critical
```suggestion
    if ssl[:cacerts] || ssl[:cacertfile] do
      config
    else
      %{config | ssl: Keyword.put(ssl, :cacerts, Certificate.ca_certs())}
    end
```